ml/base/config.py

```diff
@@ -38,6 +39,27 @@ def from_dict(cls, dic):
     def from_internal_file(cls, config_name):
         pass
 
+    @classmethod
+    def override_with_flag(cls, args_string):
+	args_list=args_string.split('--')
+	
+	#Go through all the flags.
```

Comment:
This is not a very constructive comment... normally a for loop is pretty clear for what it's doing, also spacing needed after `#`